app/javascript/tinymce-handler.js

```diff
@@ -0,0 +1,91 @@
+const uploadHandler = async function (blobInfo, progress) {
+
+  // First, use the back end to generate a presigned URL
+  const data = await fetch("/get_presigned_url", {
+    method: "GET",
+  });
+  const json = await data.json();
+  const { presigned_url } = json;
+  
+  return new Promise(async (resolve, reject) => {
+
+    try {
+      const { url } = await fetch(presigned_url, {method: 'PUT', body: blobInfo.blob()});
+      resolve(url.split("?")[0]) 
+    } catch (err) {
+      reject({
+        message: `Image upload failed: ${err.message}`,
+        remove: true
+      })
+    }
+  });
+};
+
+window.addEventListener("turbo:load", async (event) => {
+  tinymce.remove(); // Important! Turbo sometimes skips this code if it thinks TinyMCE has been init-ed already.
+  // Remove and init to force it to render the editor every time you load a new page.
+  tinymce.init({
+    selector: ".tinymce",
+    height: 500,
+    menubar: false,
+    plugins: [
+      "a11ychecker",
+      "advlist",
+      "advcode",
+      "advtable",
+      "autolink",
+      "checklist",
+      "export",
+      "lists",
+      "link",
+      "image",
+      "charmap",
+      "preview",
+      "anchor",
+      "searchreplace",
+      "visualblocks",
+      "powerpaste",
+      "fullscreen",
+      "formatpainter",
+      "insertdatetime",
+      "media",
+      "table",
+      "help",
+      "wordcount",
+    ],
+    toolbar:
+      "undo redo | casechange blocks | bold italic backcolor | " +
+      "alignleft aligncenter alignright alignjustify | image | " +
+      "bullist numlist checklist outdent indent | removeformat | a11ycheck code table help",
+    image_title: true,
+    file_picker_types: "image",
```

Comment:
Where does the input get created on the page? Doest it just get place anywhere? This would make it hard to style/change later?